cdk/apps/htsget/requirements.txt

```diff
@@ -0,0 +1,11 @@
+aws-cdk.core
```

Comment:
Is there a particular reason why we are placing the dependencies here?
In other apps we placed them in setup.py following the CDK template. Is that no longer recommended?

Reply:
No particular reason. Happy to move it to setup tool in next, yes.